lib/samples/apply_scheduled_updates_to_preplanned_map_area/apply_scheduled_updates_to_preplanned_map_area_sample.dart

```diff
@@ -30,80 +31,163 @@ class ApplyScheduledUpdatesToPreplannedMapAreaSample extends StatefulWidget {
   var _updateSizeKB = 0.0;
-  OfflineMapSyncTask? _offlineMapSyncTask;
+  // The Active mobile map package
   MobileMapPackage? _mobileMapPackage;
+  // Offline task and parameters used for updating the map package
+  OfflineMapSyncTask? _offlineMapSyncTask;
   OfflineMapSyncParameters? _mapSyncParameters;
+  // The location of the map package on the device
   late final Uri _dataUri;
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
-        child: Column(
-          crossAxisAlignment: CrossAxisAlignment.center,
+        top: false,
+        child: Stack(
           children: [
-            Expanded(
-              child: ArcGISMapView(
-                controllerProvider: () => _mapViewController,
-                onMapViewReady: onMapViewReady,
-              ),
-            ),
-            SizedBox(
-              height: 75,
-              child: Row(
-                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                crossAxisAlignment: CrossAxisAlignment.center,
-                children: [
-                  Column(
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                    children: [
-                      Text('Updates: ${_updateStatus.name.toUpperCase()}'),
-                      Text('Update Size: ${_updateSizeKB}KB'),
-                    ],
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                Expanded(
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
                   ),
-                  Center(
-                    child: ElevatedButton(
-                      onPressed: _canUpdate ? syncUpdates : null,
-                      child: const Text('Update'),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: [
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: [
+                        Text('Updates: ${_updateStatus.name.toUpperCase()}'),
+                        Text('Update Size: ${_updateSizeKB}KB'),
+                      ],
                     ),
-                  ),
-                ],
+                    Center(
+                      child: ElevatedButton(
+                        onPressed: _updating
+                            ? null
+                            : _canUpdate
+                                ? syncUpdates
+                                : resetMapPackage,
+                        child: _canUpdate
+                            ? const Text('Update')
```

Comment:
```suggestion
                            ? const Text('Apply Updates')
```
Per the readme "How to use the sample" section